sdk/tests/conformance2/extensions/ext-texture-norm16.html

```diff
@@ -153,7 +153,8 @@
 
   wtu.glErrorShouldBe(gl, gl.NO_ERROR, "texture and framebuffer setup succeed");
 
-  wtu.setupTexturedQuad(gl, 0, 1, wtu.simpleHighPrecisionTextureFragmentShader);
```

Comment:
I would like to see the ESSL100 test retained, but testing for the result within a tolerance. It's important to test that ESSL100 shaders sample from this correctly, and ideally that RTT works within the restrictions of mediump.